crates/red_knot_python_semantic/resources/primer/good.txt

```diff
@@ -1,8 +1,6 @@
-arrow
```

Comment:
This package is not included upstream. I used it for testing something, I don't think it's important that we keep it.